src/components/Kv/KvPillToggle.vue

```diff
@@ -1,14 +1,20 @@
 <template>
-	<div>
-		<div
-			v-for="option in options" :key="option.key"
+	<div class="pill-toggle">
+		<span
+			v-for="(option, index) in options" :key="option.key"
 			@click="pillToggled(option.key)"
-			class="pill-option"
-			:class="{ active: selected === option.key }">
-			<h3>
+			:class="[
+				'pill-option',
```

Comment:
Let's use `first-child` and `last-child` for this instead of array indices

Reply:
@ksekhri PR updated with `first-child` and `last-child` and other minor SASSy changes.